addons/sale/views/sale_views.xml

```diff
@@ -288,7 +288,11 @@
                         </group>
                         <group>
                             <field name="validity_date" attrs="{'invisible': [('state', 'in', ['sale', 'done'])]}"/>
-                            <field name="confirmation_date" attrs="{'invisible': [('state', 'in', ['draft', 'sent', 'cancel'])], 'required': [('state', 'in', ['sale', 'done'])]}"/>
+                            <div class="o_td_label"> 
+                                <label for="create_date" string="Quotation Date" attrs="{'invisible': [('state', 'in', ['sale', 'done', 'cancel'])]}"/> 
```

Comment:
Don't refer to create_date for the date_order label. The string of the label can change, but it is confusing if we show the information of a field for another field when hovering on it.

```xml
<div class="o_td_label">
    <label for="date_order" string="Quotation Date" attrs="{'invisible': [('state', 'in', ['sale', 'done', 'cancel'])]}"/>
    <label for="date_order" string="Order Date" attrs="{'invisible': [('state', 'in', ['draft', 'sent', 'cancel'])]}"/>
</div>
```

Reply:
changes are done.